setup.py

```diff
@@ -63,8 +63,11 @@ def get_requirements(file: Path) -> List[str]:
     requirements = []
     with file.open("r") as fh:
         for line in fh.readlines():
-            if line.startswith("#"):
+            if line.startswith("#") or line.strip() == "":
```

Comment:
It seems that we are doing the same strip operation on the same variable in three different places. What about `_line = line.strip()` right after L65, so we can then remove `strip()` from L66, L69 and L72?

Reply:
No the strip operations are not identical.

Btw. don't name local variables `_...` (I saw this multiple times in your code.)
These are private members of classes.